### DISCUSSION TITLE
Create a sensor that gives me a monthly running total of green energy

### DISCUSSION THREAD

**Discussion by stephencrowther** [CJNE/ha-myenergi#604] [Q&A]
Hello can anyone guide me on creating a sensor that takes the daily green energy used sensor directly from this integration and gives me an accumulative sensor (so it adds up each days value over a month and resets every month).

you can see this figure on the eddi device directly but it isn’t exposed by the API I guess to home assistant.

thanks

**Comment by HLFCode** [comment]
Have a look at the [Utility Meter](https://www.home-assistant.io/integrations/utility_meter/) integration
